packages/subgraph/schema.graphql

```diff
@@ -888,14 +888,23 @@ type MultisigPlugin implements IPlugin @entity {
 }
 
 type MultisigApprover @entity {
+  """
+  Current Approver of the multisig
+  """
   id: ID! # plugin_address + member_address
-  address: String # address as string to facilitate filtering by address on the UI
-  proposals: [MultisigProposalApprover!]! @derivedFrom(field: "approver")
+  address: Bytes!
```

Comment:
`Bytes` are shown as a string in subgraph and can be used for filtering, all the other addresses in the schema are as Bytes, I dindt see the point of  leaving it as a string

Reply:
I see no problem with having them as Bytes. my main concern is that they explicitly say `address as string to facilitate filtering by address on the UI`, not sure how correct that comment is but would like to know. 

This is also being done in other places ([here](https://github.com/aragon/osx/blob/c27df037cfb99ad20d9c23a04f20db8f9eb73379/packages/subgraph/schema.graphql#L694), [here](https://github.com/aragon/osx/blob/c27df037cfb99ad20d9c23a04f20db8f9eb73379/packages/subgraph/schema.graphql#L841), [here](https://github.com/aragon/osx/blob/c27df037cfb99ad20d9c23a04f20db8f9eb73379/packages/subgraph/schema.graphql#L892))

I just want to avoid having to affect the APp team in case the comment is correct and is placed as a string to make their work easier.

Reply:
Yep, that makes sense, I personally see no difference in behaivour, but lets ask @Rekard0 who is who added the comment in the first place

Reply:
we checked and there is no reason for having it as a string, we also considered that we should unify all the addresses to bytes.